elements/rh-footer/README.md

```diff
@@ -11,6 +11,10 @@ npm install @rhds/elements
 import '@rhds/elements/rh-footer/rh-footer.js';
 ```
 
+```html
+<link rel="stylesheet" href="https://unpkg.com/@rhds/elements/elements/rh-footer/rh-footer-lightdom.css" />
```

Comment:
Adding example for how to pull in css file off of a CDN

Reply:
I'm thinking about adding a prepublishOnly step which copies all the `/elements/*/*-lighdom.css` to `$1/$2-lightdom.css`. Doesn't fill me with joy though

Reply:
https://github.com/RedHat-UX/red-hat-design-system/issues/418